manifests/04_cluster-ingress-operator.yaml

```diff
@@ -21,7 +21,7 @@ spec:
             name: metrics
           command:
           - cluster-ingress-operator
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
If the default is a `latest` tag, would `Always` be more convenient for development?

Reply:
Yes, helps in development but I don't think we want to ship with 'Always' image pull policy.

Reply:
Good point, let's remove the key